examples/Model-class-OneParExpExpPrior.R

```diff
@@ -0,0 +1,5 @@
+my_model <- OneParExpExpPrior(
+  skel_probs = seq(from = 0.1, to = 0.9, length = 5),
```

Comment:
Maybe better to manualy create these values rather than using `seq`, so `c(0.1, 0.3, 0.5, 0.7, 0.9)`.